src/components/featured-card/styles.ts

```diff
@@ -53,7 +53,7 @@ const FeaturedCardContent: ThemeUICSSObject = {
 
 const FeaturedCardImageWrapper: ThemeUICSSObject = {
   order: [1, 2, null, null],
-  width: [null, '50%', null, '50%'],
+  width: [null, '50%', null, null],
   display: [null, 'flex', null, null],
   justifyContent: [null, 'center', null, null],
   marginBottom: ['inc40', 0, null, null],
```

Comment:
nitpick, but you actually don't need to specify the `null`s at the end of these array. If the array ends short, `null` is implied for the following array entries.

Reply:
great to know, my next PR will be updating `<FeaturedCard />` styles so I'll incorporate it there